### PR TITLE
AppsTest list enable and disable

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
This PR adds tests for unauthorized apps tests.

### CODE REVIEW

File: tests/unauthorized/appsTest.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ describe('Unauthorized: Currently testing apps management,', function () {
     oc.login()
   })
 
-  it('checking method : getApps', function (done) {
+  fit('checking method : getApps', function (done) {
```
Comment (individual-it) [inline tests/unauthorized/appsTest.js:27]: I believe its enough to have `fdescribe` on the top if you want to enable all the tests

File: tests/karma.js [hunk 2]
```diff
@@ -556,6 +556,108 @@ beforeAll(function (done) {
             '</ocs>\n'
         }
       }))
+    .then(() =>
+      provider.addInteraction({
+        uponReceiving: 'apps GET request',
```
Comment (individual-it) [inline tests/karma.js:561]: please rename, to make clear its a request with an invalid auth

**Review by individual-it — CHANGES_REQUESTED** [review]

File: tests/unauthorized/appsTest.js (hunk 1 above)
Comment (jasson99) [reply tests/unauthorized/appsTest.js]: fixed

File: tests/karma.js (hunk 2 above)
Comment (jasson99) [reply tests/karma.js]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**